content/04.discussion.md

```diff
@@ -1,3 +1,25 @@
 ## Discussion
 
-Stub in discussion section   
+Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians across the nation to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
```

Comment:
"across the nation" 

Do we have international collaborators?

Reply:
oh good point - we do have clinicians outside - was thinking of those who worked on GitHub specifically, but I can update!